src/routes/klageskjema/begrunnelse/begrunnelse.tsx

```diff
@@ -191,17 +193,34 @@ const Begrunnelse = ({ klage }: Props) => {
                 </KlageAlertStripeFeil>
             )}
 
-            <Reasons checkedReasons={reasons} setCheckedReasons={setReasons} />
-
             <Section>
-                <KlageUndertittel>Vedtaksdato (valgfri)</KlageUndertittel>
-                <Datepicker
-                    onChange={(dateISO, isValid) => setVedtakDate(isValid ? dateISO : null)}
-                    value={vedtakDate ?? undefined}
-                    showYearSelector
-                    limitations={{
-                        maxDate: new Date().toISOString().substring(0, 10)
-                    }}
+                <KlageUndertittel>Hva er du uenig i?</KlageUndertittel>
+
+                <Row>
+                    <Reasons checkedReasons={reasons} setCheckedReasons={setReasons} />
+                </Row>
+                <Row>
+                    <label htmlFor="vedtaksdato">
```

Comment:
I stedet for å pakke `FieldLabel` (som er en `<p>`) inn i en `<label>`, kan vi gjøre om `FieldLabel` om til å rendre en `<label>` og stjele stylingen for `<label>` fra `Input`?

Reply:
> I stedet for å pakke `FieldLabel` (som er en `<p>`) inn i en `<label>`, kan vi gjøre om `FieldLabel` om til å rendre en `<label>` og stjele stylingen for `<label>` fra `Input`?

Tror vi kan bare bruke Label-komponenten fra designsystemet? Har oppdatert med den.

Reply:
Perfekt :)
Satser på at det er den som er brukt i `Input` og derfor ser lik ut.

Reply:
Har du testet at det ser riktig ut?

Reply:
> Har du testet at det ser riktig ut?

![image](https://user-images.githubusercontent.com/19277989/101740897-b4bc2a00-3ac9-11eb-80af-0dd7f9b4967c.png)

Reply:
Helt lik CSS :)

Reply:
Perfekt, kjørte det opp lokalt her også for å se og trykke.
Det både ser og fungerer bra :)